docstr_coverage/coverage.py

```diff
@@ -306,7 +310,11 @@ def print_docstring(base, node, filename, ignore_names=()):
     log("Total docstring coverage: %.1f%%; " % (total_results["coverage"]), 1, True)
 
     #################### Calculate Total Grade ####################
-    grade = next(_[0] for _ in GRADES if _[1] <= total_results["coverage"])
+    grade = next(
+        message
+        for message, grade_threshold in GRADES
+        if total_results["coverage"] <= grade_threshold
```

Comment:
Looks like these two got switched. Nice clarification, though!
```suggestion
        if grade_threshold <= total_results["coverage"]
```

Reply:
Ah no, actually this was the source of the bug, that's why I switched these. This way the minimal grade_threshold would be `1`, while `total_results["coverage"]` can become `0`.

Reply:
Ahh I see. What if we make the suggested change, then also change the last entry in `GRADES` to be `("Do you even docstring?", 0),`. This should take care of the bug you mentioned while also keeping the message accurate. In its current state, the "Amazing" message always gets printed.

Reply:
what a wonderful safety net I've built 🤦 thanks!